cmd_main.cpp

```diff
@@ -0,0 +1,128 @@
+#include "cmd.h"
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include <grpc/grpc.h>
+#include <grpcpp/channel.h>
+#include <grpcpp/client_context.h>
+#include <grpcpp/create_channel.h>
+#include <grpcpp/security/credentials.h>
+
+int main(int argc, char const *argv[])
```

Comment:
According to spec, `gflags` should be used instead of manually parsing the the arguments.